test/NuGet.Core.Tests/NuGet.Configuration.Test/TestUtils.cs

```diff
@@ -18,5 +20,19 @@ public static void CreateConfigurationFile(string configurationPath, string mock
                 file.Write(info, 0, info.Count());
             }
         }
+
+        public static byte[] GetFileHash(string fileName)
+        {
+            var hashAlgorithm = SHA512.Create();
```

Comment:
`HashAlgorithm` implements `IDisposable`.  You need to wrap this with a `using` statement.